app/src/main/java/live/hms/app2/ui/meeting/pinnedvideo/PinnedVideoFragment.kt

```diff
@@ -122,13 +125,12 @@ class PinnedVideoFragment : Fragment() {
     crashlyticsLog(TAG, "handleOnPinVideoVisibilityChange: isViewVisible=${isViewVisible}")
 
     pinnedTrack?.let { track : MeetingTrack ->
-      binding.pinVideo.surfaceView.apply {
+      binding.pinVideo.hmsVideoView.apply {
         if (isViewVisible) {
-          SurfaceViewRendererUtil.bind(this, track).let { success ->
-            if (success) visibility = if (track.video?.isDegraded == true) View.INVISIBLE else View.VISIBLE
-          }
+          track.video?.let { addTrack(it) }
+          visibility = if (track.video?.isDegraded == true) View.INVISIBLE else View.VISIBLE
```

Comment:
`visibility = if (track.video?.isDegraded == true) View.INVISIBLE else View.VISIBLE` 
This should be done in the not null part of `track.video`